### DISCUSSION TITLE
Do you need a Pro account if you are hosting expose on your own server?

### DISCUSSION THREAD

**Discussion by abishekrsrikaanth** [exposedev/expose#314] [Q&A]
Do you need a Pro account if you are hosting expose on your own server?

**Comment by mpociot** [comment]
No, as Expose is open source you can simply self host it. Or use Expose Pro and use your own domain through our infrastructure

**Comment by abishekrsrikaanth** [comment]
@mpociot since I was not able to use expose as it is currently restricted through Cloudflare in India, I self-hosted it, but it's definitely slower. Are there any minimum server requirements when it comes to self-hosting it?
